lib/plugins/TaskRunner/LAMPApp.js

```diff
@@ -70,7 +70,7 @@ class LAMPApp extends Script {
 
   addScriptHeader() {
     this.script = this.script.concat([
-      'sleep 3',
+      'READY=0; while ! `nc -z 127.0.0.1 3306` && [ $READY -lt 60 ]; do echo "Waiting for MySQL..."; READY=$((READY + 1)); sleep 1; done; if `nc -z 127.0.0.1 3306`; then echo "MySQL is ready."; else echo "MySQL failed to start!"; exit 1; fi;',
```

Comment:
While I agree, right now the port is baked into the docker image that we are using, so that can't be controlled from here unless you do a bunch of tricky stuff anyway.